301-multi-vmss-linux/nestedtemplates/base.json

```diff
@@ -27,38 +24,39 @@
   "variables": {
     "imagePublisher": "Canonical",
     "imageOffer": "UbuntuServer",
-    "imageVersion": "15.10",
+    "osVersion": "16.04.0-LTS",
     "subnetName": "subnet",
     "storageAccountType": "Standard_LRS",
     "vhdContainerName": "vhds",
     "uniqueStringArray": [
-      "[concat(uniqueString(concat(resourceGroup().id, parameters('newStorageAccountPrefix'), '0')))]",
-      "[concat(uniqueString(concat(resourceGroup().id, parameters('newStorageAccountPrefix'), '1')))]",
-      "[concat(uniqueString(concat(resourceGroup().id, parameters('newStorageAccountPrefix'), '2')))]",
-      "[concat(uniqueString(concat(resourceGroup().id, parameters('newStorageAccountPrefix'), '3')))]",
-      "[concat(uniqueString(concat(resourceGroup().id, parameters('newStorageAccountPrefix'), '4')))]"
+      "[concat(uniqueString(resourceGroup().id, '0'), '0')]",
+      "[concat(uniqueString(resourceGroup().id, '1'), '1')]",
+      "[concat(uniqueString(resourceGroup().id, '2'), '2')]",
+      "[concat(uniqueString(resourceGroup().id, '3'), '3')]",
+      "[concat(uniqueString(resourceGroup().id, '4'), '4')]"
     ],
-    "saCount": "[length(variables('uniqueStringArray'))]",
-    "computeApiVersion": "2016-03-30",
-    "storageApiVersion": "2015-06-15"
+    "saCount": "[length(variables('uniqueStringArray'))]"
```

Comment:
No need to have an saCount variable - you could just inline the call to [length(variables('uniqueStringArray'))]